src/java.base/share/classes/java/time/OffsetDateTime.java

```diff
@@ -1801,7 +1802,10 @@ public long toEpochSecond() {
      * consistent with {@code equals()}.
      *
      * @param other  the other date-time to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value, that is the comparison with the {@code other}'s instant, if they are not equal;
```

Comment:
This text is incorrect. It does not match the description above re offsets. (This would be a suitable description for `compareInstant`)

Reply:
Line 1806 describes the comparison of the local time if the instants are equal.
```and if equal to the {@code other}'s instant, the comparison of the {@code other}'s local date-time```